Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.h

```diff
@@ -45,9 +48,17 @@ enum class WiimoteGroup
   Swing,
   Rumble,
   Attachments,
-
   Options,
-  Hotkeys
+  Hotkeys,
+  IMUAccelerometer,
+  IMUGyro,
+  IMUPoint,
+};
+
+enum class MotionSource
+{
+  Emulated,
+  IMU
```

Comment:
Whats an `IMU`? This feels like it wouldn't be a problem to just spell it out (or at least document with a comment instead if you think this is a well-known abbreviation and I just don't know it).
This also goes for other occurrences unless you go the comment route.

Also, trailing comma please.

Reply:
I also do not know what IMU means.

Reply:
At first I spelled it out as InertialMeasurementUnit but it just looked so damn ugly. I think it's a reasonably well known abbrevation and describes the thingy perfectly, so I'll just add a comment. Comma will be fixed. From Wikipedia: 

> An inertial measurement unit (IMU) is an electronic device that measures and reports a body's specific force, angular rate, and sometimes the orientation of the body, using a combination of accelerometers, gyroscopes, and sometimes magnetometers.

Reply:
Why not just `Emulated` and `Real`?

Reply:
`Emulated` and `Real` is what I used in the GUI. But I envision future support for other motion sources that do not present raw accelerometer and gyro data, but perhaps position and angle directly. That would be a second `Real` motion source.